docs/remote-control/operating-the-remote.md

```diff
@@ -94,13 +108,17 @@ an active playback.
 
 
 
-## Recording Palettes
+## Recording Palettes, Groups and Cues
```

Comment:
Somewhere in this section it should probably be mentioned that when recording playbacks the consoles record mode is followed.